great_expectations/core/usage_statistics/anonymizers/store_anonymizer.py

```diff
@@ -40,13 +47,13 @@ def _anonymize_store_backend_info(
         assert (
             store_backend_obj or store_backend_object_config
         ), "Must pass store_backend_obj or store_backend_object_config."
-        anonymized_info_dict = {}
+        anonymized_info_dict: dict = {}
         if store_backend_obj is not None:
             self._anonymize_object_info(
                 object_=store_backend_obj,
                 anonymized_info_dict=anonymized_info_dict,
             )
-        else:
+        elif store_backend_object_config is not None:
```

Comment:
runtime change here.
This block would fail with an attribute error if this condition was not met, so I believe it's a safe change.

Reply:
Actually, I could make this more explicit.